org.kde.discover.yaml

```diff
@@ -0,0 +1,332 @@
+        - -Dbash_completion=false
+        - -Dman=false
+        - -Dsystemd=disabled
+        - -Dpassim=disabled
+        - -Ddocs=disabled
+        - -Dtests=false
+        - -Dvendor_ids_dir=/app/share/hwdata
+        - -Dvalgrind=disabled
+        - -Dintrospection=disabled
+      sources:
+        - type: archive
+          url: https://github.com/fwupd/fwupd/releases/download/2.0.16/fwupd-2.0.16.tar.xz
+          sha256: 79cce6b61d9ea531de0ef3b13e7f2682c699901b11f410b2ed91192618e1620e
+          x-checker-data:
+            type: anitya
+            project-id: 5833
+            url-template: https://github.com/fwupd/fwupd/releases/download/$version/fwupd-$version.tar.xz
+
+      modules:
+      - jinja2.yaml
+
+      - name: libjcat
+        buildsystem: meson
+        config-opts:
+          - -Dintrospection=false
+          - -Dgtkdoc=false
+          - -Dtests=false
+          - -Dcli=false
+          - --sysconfdir=/app/etc
+          - --localstatedir=/var/data
+        sources:
+          - type: archive
+            url: https://github.com/hughsie/libjcat/releases/download/0.2.5/libjcat-0.2.5.tar.xz
+            sha256: 066e402168c51bffddcf325190e5901402b266fbda2a4eed772fd06a88b941bf
+            x-checker-data:
+              type: anitya
+              project-id: 229583
+              url-template: https://github.com/hughsie/libjcat/releases/download/$version/libjcat-$version.tar.xz
+
+    - name: flatpak
+      buildsystem: meson
+      config-opts:
+        - -Dgtkdoc=disabled
+        - -Ddocbook_docs=disabled
+        - -Dtests=false
+        - -Dman=disabled
+        - -Dseccomp=disabled
+        - -Dselinux_module=disabled
+        - -Dmalcontent=disabled
+        - -Dsandboxed_triggers=false
+        - -Dsystem_helper=enabled
+        - -Dhttp_backend=curl
+        - -Dsystemd=disabled
+        - -Dsystem_install_dir=/var/lib/flatpak
+        - -Dsystem_bubblewrap=bwrap
+        - -Dsystem_dbus_proxy=xdg-dbus-proxy
+        - --sysconfdir=/var/run/host/etc
+      cleanup:
+        - /bin/flatpak-bisect
+        - /bin/flatpak-coredumpctl
+        - /etc/profile.d
+        - /lib/systemd
+        - /share/dbus-1/interfaces/org.freedesktop.*
+        - /share/dbus-1/services/org.freedesktop.*
+        - /share/flatpak/triggers
+        - /share/gdm
+        - /share/fish
+        - /share/zsh
+        - /share/gir-1.0
+        - /share/bash-completion
+        - /lib/girepository-1.0
+        - /lib/sysusers.d
+        - /lib/tmpfiles.d
+      post-install:
+        - install -Dpm0755 -t /app/bin /usr/bin/update-{mime,desktop}-database
+      sources:
+        - type: git
+          url: https://github.com/flatpak/flatpak/
+          commit: b676905d91aa3c4e524ca663784d4cf085c465d7
+          tag: 1.16.1
+          x-checker-data:
+            type: git
+            tag-pattern: "^([\\d.]+)$"
+
+      modules:
+         - shared-modules/intltool/intltool-0.51.json
+         - name: polkit
```

Comment:
If you [disable the system helper](https://github.com/flatpak/flatpak/blob/bd468a89596e6f89a364b322e42a70948672f4e5/meson.build#L189) polkit is no loner needed